Batch 6: Pretty Momma

ericsorenson opened the pull request:
- 8707 [Court: Content Changes for SPTO/Pretrial Memo - Chambers Phone # - Release with 8662 and 8663](https://github.com/flexion/ef-cms/issues/8707) 
- 8684 [Court: New Document Needed To Support Remote Proceedings](https://github.com/flexion/ef-cms/issues/8684) 
- 8663 [Court: Content Changes for SPTO for Small Cases - Release with 8662](https://github.com/flexion/ef-cms/issues/8663) 
- 8662 [Court: Content Changes for SPTO (Regular Cases) - Release with 8663](https://github.com/flexion/ef-cms/issues/8662) 
- 8652 [Court: Content Changes Needed to Notice of Receipt of Petition](https://github.com/flexion/ef-cms/issues/8652) 
- 8651 [Order Search: Special Character Usage ](https://github.com/flexion/ef-cms/issues/8651) 
- 8650 [Order Search: Case Caption and Docket# section of DAWSON-generated/created content should be searchable content](https://github.com/flexion/ef-cms/issues/8650) 
- 8553 [Usability Testing: Order Search Round ustaxcourt/ef-cms#6492](https://github.com/flexion/ef-cms/issues/8553)

codyseibert committed:
8670 - fixing migration

ericsorenson committed:
Merge pull request #8661 from flexion/from-staging

From ustaxcourt/staging

mrinsin committed:
8650 fixing tests and re adding mappings for docket entry

matthopson committed:
Merge pull request #8673 from flexion/8670-using-trim-horizon

8670 - using trim horizon

rachaelparris committed:
Merge pull request #8671 from flexion/devex-864

Devex 864: reducing size of unit test files

rosiehsmith committed:
Merge pull request #8672 from flexion/8650-story

8650: add docket number and case caption to indexed document contents

klohman committed:
Update ustc-story-template.md

rosiehsmith committed:
Merge branch 'develop' of https://github.com/flexion/ef-cms into devex-971

rachaelparris committed:
Devex 864: reducing size of unit test files

rachaelparris committed:
Devex 864: more test file cleanup

ericsorenson committed:
maintenance: upgrade cypress

rosiehsmith committed:
devex-971: a little more coverage

codyseibert committed:
remove records result in empty objects trying to be processed

rachaelparris committed:
Devex 864: fix tests

rachaelparris committed:
Devex 864: use foreach instead of map

rachaelparris committed:
Merge pull request #8675 from flexion/devex-864

Devex 864: more test file cleanup

codyseibert committed:
Merge pull request #8678 from flexion/issue-with-global-filter

Issue with the live migrations crashing on REMOVE events

rosiehsmith committed:
devex-971: pr feedback cleanup

rosiehsmith committed:
Merge pull request #8677 from flexion/devex-971

Devex 971

codyseibert committed:
Merge branch 'staging' into back-merge-staging

ericsorenson committed:
Merge pull request #8680 from flexion/experimental5

Maintenance: Upgrade Cypress

ericsorenson committed:
maintenance: update pdfjs-dist

rachaelparris committed:
Devex 864: working on test cleanup

rachaelparris committed:
8650: migration to parse older orders [skip ci]

rosiehsmith committed:
8650: wip working on migration [skip ci]

mrinsin committed:
8650 finishing up migration script

rachaelparris committed:
8650: working on tests [skip ci]

ericsorenson committed:
try not minifying

klohman committed:
Update UX-Documentation.md

rosiehsmith committed:
8650: wip tests [skip ci]

klohman committed:
Add files via upload

klohman committed:
Create UX-Documentation.md

mrinsin committed:
8650 adding tests [skip ci]

ericsorenson committed:
Merge pull request #8681 from flexion/back-merge-staging

Back Merge Staging

rosiehsmith committed:
8650: wip [skip ci]

mrinsin committed:
8650 [skip ci] trying things

rosiehsmith committed:
8650: wip working on migration [skip ci]

ericsorenson committed:
Merge pull request #8683 from flexion/experimental5

Maintenance: Update pdfjs-dist, skip lambda minification

TomElliottFlexion committed:
8662: Updated Standing Pretrial order for regular cases, as well as the preparation checklist

rachaelparris committed:
Devex 864: cleaning up more unit tests

rachaelparris committed:
Devex 864: more unit test cleanup/splitting into separate files

rachaelparris committed:
put max-lines limit back

TomElliottFlexion committed:
8662:  Minor formatting changes

ericsorenson committed:
maintenance: upgrade puppeteer

rachaelparris committed:
8650: working on tests [skip ci]

TomElliottFlexion committed:
8662: Added back docket header

rosiehsmith committed:
8650: finished tests for migration [skip ci]

rosiehsmith committed:
8650: cleanup

ericsorenson committed:
maintenance: reset deps

rachaelparris committed:
Merge pull request #8686 from flexion/devex-864

Devex 864: unit test cleanup

mrinsin committed:
[skip ci] wip adding env vars to tf

ericsorenson committed:
Merge branch 'staging' into staging-to-develop

rosiehsmith committed:
8650: updating tf type stuff for migration [skip ci]

mrinsin committed:
trying things

TomElliottFlexion committed:
8663: Update formatting, fix phone number [skip ci]

rachaelparris committed:
8650: fix environment variables for migration lambda

rachaelparris committed:
Merge branch 'develop' into 8650-parse-old-orders

ericsorenson committed:
Merge pull request #8691 from flexion/staging-to-develop

flexion/staging to flexion/develop

rachaelparris committed:
8650: working on migration deploy

rachaelparris committed:
don't run tests on exp1

TomElliottFlexion committed:
8663: Formatting

TomElliottFlexion committed:
8663: Fixed test

codyseibert committed:
8684 - putting motion behind a flag

ericsorenson committed:
lock chrome-aws-lambda / puppeteer to version 10.0.0

TomElliottFlexion committed:
8663: Increased font size and fixed duplicate memos

TomElliottFlexion committed:
8663: Increased font size and reduced line-height

TomElliottFlexion committed:
Merge branch '8662-regular-pretrial-order' of https://github.com/flexion/ef-cms into 8663-small-case-pretrial

mrinsin committed:
adding documents bucket name to config yml

mrinsin committed:
Merge branch '8650-parse-old-orders' into experimental1

codyseibert committed:
8684 - adding tests

TomElliottFlexion committed:
Merge pull request #8687 from flexion/8662-regular-pretrial-order

8662: regular pretrial order formatting

TomElliottFlexion committed:
Merge pull request #8694 from flexion/8663-small-case-pretrial

8663: small case pretrial order formatting

mrinsin committed:
adding env var to migration_segments

rosiehsmith committed:
Merge branch 'experimental1' of https://github.com/flexion/ef-cms into experimental1

mrinsin committed:
mergin updates

matthopson committed:
8652 - Template for updated notice of receipt of petition

matthopson committed:
8652 - add updated notice of petition template generator

matthopson committed:
Revert "consolidate getChromiumBrowser mocks"

This reverts commit c81a79928562a9b2a678c934de76b07816c4b299.

matthopson committed:
8652 - Add real chromium browser to notice of petition

matthopson committed:
8652 - Add chromium browser to notice of trial issued test

codyseibert committed:
8684 - pr comments

matthopson committed:
8652 - Display fixes

mrinsin committed:
removing unused var

mrinsin committed:
removing unused var

codyseibert committed:
Merge pull request #8695 from flexion/8684-support-remove-proceedings

8684 - putting motion behind a flag

matthopson committed:
Merge pull request #8696 from flexion/8652-notice-of-receipt

8652 - Updated Notice of Receipt

ericsorenson committed:
Merge pull request #8688 from flexion/experimental5

Maintenance: Upgrade puppeteer

ericsorenson committed:
maintenance: minor version updates

klohman committed:
Update UX-Documentation.md

mrinsin committed:
8650 adding missing envs

mrinsin committed:
Merge branch 'experimental1' into 8650-parse-old-orders

ericsorenson committed:
Merge pull request #8698 from flexion/experimental5

Maintenance: Minor version updates

TomElliottFlexion committed:
8663: Formatting for feedback

TomElliottFlexion committed:
8862: Formatting feedback

rachaelparris committed:
fix environment variable issue

rosiehsmith committed:
Merge branch 'experimental1' of https://github.com/flexion/ef-cms into experimental1

rosiehsmith committed:
Merge branch 'experimental1' of https://github.com/flexion/ef-cms into experimental1

TomElliottFlexion committed:
Merge pull request #8701 from flexion/8662-pretrial-order-feedback

8662: Pretrial order feedback

rosiehsmith committed:
8650: adding puppeteer layer for migration segments

TomElliottFlexion committed:
Merge pull request #8700 from flexion/8663-small-case-pretrial

8663: Formatting for feedback

TomElliottFlexion committed:
8662: De-capitalize word

rachaelparris committed:
fix another env var issue

codyseibert committed:
including pdfjs in the migration zip files

TomElliottFlexion committed:
Merge pull request #8704 from flexion/8662-pretrial-order-feedback

8662: De-capitalize word

TomElliottFlexion committed:
8663: Add spaces

rosiehsmith committed:
using validateForMigration to get better logging since only 0000 migration fails at this point

TomElliottFlexion committed:
Merge pull request #8706 from flexion/8663-small-case-pretrial

8663: Add spaces

kkoskelin committed:
devex 973: 100% coverage for test:api task

rachaelparris committed:
Devex 864: continue reducing size of unit test files

rachaelparris committed:
Devex 864: remove redundant tests

kkoskelin committed:
Merge pull request #8709 from flexion/devex-web-api-100coverage

devex 973: 100% coverage for test:api task

ericsorenson committed:
Merge branch 'staging' into staging-to-develop

rosiehsmith committed:
Merge branch 'experimental1' into 8650-parse-old-orders

rosiehsmith committed:
8650: wip integration tests for test cases [skip ci]

codyseibert committed:
8650 - helper file to update the case caption

rachaelparris committed:
8650: working on tests [skip ci]

TomElliottFlexion committed:
8707: Add chambers phone number

TomElliottFlexion committed:
8707: Add phone number to details

rosiehsmith committed:
8650: wip tests [skip ci]

Commit:
8707: display the formatted chambers phone number on trial session detail page

codyseibert committed:
8650 - verify the orders return in the query

ericsorenson committed:
Merge pull request #8712 from flexion/staging-to-develop

Staging to develop

rachaelparris committed:
8650: working on tests [skip ci]

Commit:
8707: chambers phone number validation logic with remote/in person

rosiehsmith committed:
8650: done with test scenario 6 wip [skip ci]

codyseibert committed:
8650 - test case 7 [skip ci]

TomElliottFlexion committed:
8707: Updated tests

codyseibert committed:
8650 - test case 8 [skip ci]

TomElliottFlexion committed:
8707: Updated unit test for interactor

rosiehsmith committed:
8650: started docket number tests [skip ci]

rosiehsmith committed:
8650: fixing test setup [skip ci]

TomElliottFlexion committed:
8707: Display 'no phone number' for in person trial sessions

TomElliottFlexion committed:
8707: Set chambers phone number on integration tests

rosiehsmith committed:
8650: wip test case 10 [skip ci]

codyseibert committed:
8650 - skip case 11 and test case 12 and 13 [skip ci]

TomElliottFlexion committed:
8707: More trial session unit tests

rachaelparris committed:
8650: working on tests [skip ci]

TomElliottFlexion committed:
8707: Add chambers phone numbers for smoketests

rosiehsmith committed:
8650: final touches on tests

rachaelparris committed:
Merge branch 'develop' into 8650-parse-old-orders

rachaelparris committed:
Merge branch '8650-parse-old-orders' of https://github.com/flexion/ef-cms into 8650-parse-old-orders

rachaelparris committed:
8650: cleanup

TomElliottFlexion committed:
8707: Fix ability to unselect judge

rachaelparris committed:
8650: lint fix

TomElliottFlexion committed:
8707: Remove comment

rachaelparris committed:
8650: fix test

TomElliottFlexion committed:
Merge pull request #8716 from flexion/8707-chambers-phone

8707: chambers phone number

rosiehsmith committed:
Merge pull request #8719 from flexion/8650-parse-old-orders

8650: parse old orders, add integration tests

kkoskelin committed:
bumping uncovered branches

ericsorenson committed:
Remove SonarCloud references

ericsorenson committed:
Merge pull request #8720 from flexion/staging-to-develop

Remove SonarCloud references

ericsorenson committed:
devex: Splat on the top of the object literals to prevent accidental overwrites

ericsorenson committed:
Merge branch 'staging' into staging-to-develop

ericsorenson committed:
Revert "devex: Splat on the top of the object literals to prevent accidental overwrites"

This reverts commit b5a956277e3a21c87db384a5d85ffc638ffe6115.

ericsorenson committed:
Merge branch 'develop' into staging-to-develop

rachaelparris committed:
Merge branch 'develop' into devex-864

kkoskelin committed:
more uncovered branches and lines in action tests

rosiehsmith committed:
Merge pull request #8721 from flexion/staging-to-develop

Staging to develop plus bonus devex

kkoskelin committed:
fix promise-related linting warnings

kkoskelin committed:
Merge pull request #8722 from flexion/devex-coverage-1.01

Devex: coverage for client unit tests

codyseibert committed:
8650 - storing contents with the docket number suffix included

codyseibert committed:
8650 - storing contents with the docket number suffix included

rachaelparris committed:
Merge branch 'develop' into devex-864

rosiehsmith committed:
8650: added test

rosiehsmith committed:
Merge pull request #8726 from flexion/8650-store-contents-with-docket-number-suffix

8650 store contents with docket number suffix

rachaelparris committed:
Devex 864: clean up addCoversheetInteractor tests

rachaelparris committed:
Merge branch 'develop' into devex-864

rachaelparris committed:
Devex 864: import file correctly

TomElliottFlexion committed:
8707: Feedback formatting

TomElliottFlexion committed:
Merge branch 'develop' of https://github.com/flexion/ef-cms into 8707-chambers-phone

rachaelparris committed:
Merge pull request #8729 from flexion/devex-864

Devex 864: clean up unit tests

TomElliottFlexion committed:
Merge pull request #8730 from flexion/8707-chambers-phone

8707: Minor formatting for feedback

rosiehsmith committed:
8650: remove docketNumber and caseCaption from documentContents in processStreamUtilities

rachaelparris committed:
Devex 864: more test file cleanup, test coverage boost

rosiehsmith committed:
8650: update integration tests to use docketNumberWithSuffix

rosiehsmith committed:
Merge pull request #8734 from flexion/8650-cleanup

8650: remove docketNumber and caseCaption from documentContents in pr…

rachaelparris committed:
Devex 864: more test file cleanup

mrinsin committed:
971 removing unecessary if

rachelschneiderman committed:
Devex 971: Adding test coverage [skip ci]

mrinsin committed:
971 adding more tests

rachelschneiderman committed:
Devex 971: Adding test coverage [skip ci]

mrinsin committed:
971 more tests

rachaelparris committed:
Merge pull request #8735 from flexion/devex-864

Devex 864: more test file cleanup + coverage

rosiehsmith committed:
971: increase coverage by changing constructor structure of AddPrivatePractitionerFactory

rosiehsmith committed:
971: updated factories and references to instantiation - SupportingDocumentInformationFactory, EditPetitionerCounselFactory, CourtIssuedDocumentFactory, AddPrivatePractitionerFactory

rosiehsmith committed:
971: updated factories and references to instantiation - ExternalDocumentFactory, ExternalDocumentInformationFactory, SecondaryDocumentInformationFactory

kkoskelin committed:
devex: fixing linting warnings about async rule

rachaelparris committed:
Devex 864: split unit tests into separate files

rachaelparris committed:
put warning back

kkoskelin committed:
Merge pull request #8737 from flexion/devex-client-coverage

devex: fixing linting warnings about async rule

rachaelparris committed:
Merge pull request #8738 from flexion/devex-864

Devex 864: split unit tests into separate files

rosiehsmith committed:
Merge branch 'develop' into 971-devex-bump-coverage

rachaelparris committed:
Devex 864: reduce size of unit test files

codyseibert committed:
devex - renaming web-client folders to have consistency

codyseibert committed:
fixing lint errors

rosiehsmith committed:
971: deepscan fixes + typo fix

kkoskelin committed:
adding missing coverage on actions

codyseibert committed:
Merge pull request #8740 from flexion/devex-directory-consistency

devex - renaming web-client folders to have consistency

rosiehsmith committed:
971: put back if statement

rosiehsmith committed:
Revert "971: deepscan fixes"

This reverts commit cfdbb4be2c941394c36559f2a58c2f8c359c5c23.

# Conflicts:
#	shared/src/business/useCases/courtIssuedDocument/generateCourtIssuedDocumentTitleInteractor.js

kkoskelin committed:
more uncovered lines now covered

mrinsin committed:
removing if check for deepscan

mrinsin committed:
more deepscan fixes

mrinsin committed:
more deepscan fixes

mrinsin committed:
more deepscan fixes pt 2

mrinsin committed:
Merge branch 'develop' of https://github.com/flexion/ef-cms into 971-devex-bump-coverage

mrinsin committed:
removing null check operator as demanded by deepscan

mrinsin committed:
devex 971 updating implementation of refactored method

rachaelparris committed:
Merge pull request #8739 from flexion/devex-864

Devex 864: reduce size of unit test files

rosiehsmith committed:
Merge pull request #8742 from flexion/971-devex-bump-coverage

971 devex bump coverage

kkoskelin committed:
Merge pull request #8743 from flexion/devex-client-coverage

devex: adding missing coverage on web-client actions

kkoskelin committed:
8650: hitting enter will execute the order search

kkoskelin committed:
Merge pull request #8745 from flexion/8650-enter-runs-search

8650: hitting enter will execute the order search

matthopson committed:
Merge pull request #8747 from ustaxcourt/staging

Backmerge Court's Staging into Batch 6 / Pretty Momma

matthopson committed:
Merge pull request #8748 from ustaxcourt/staging

Latest from Court's Staging

matthopson committed:
Merge remote-tracking branch 'origin/staging' into develop

matthopson committed:
Merge pull request #8749 from flexion/latest-from-staging-080621

Latest from Flexion Staging

rachaelparris committed:
Devex 864: reducing max file size for test files

rachaelparris committed:
Devex 864: move comment

rachaelparris committed:
Merge pull request #8750 from flexion/devex-864

Devex 864: reducing max file size for test files

rachaelparris committed:
Devex 864: more test cleanup

rachaelparris committed:
Merge branch 'develop' into devex-864

rachaelparris committed:
Merge pull request #8752 from flexion/devex-864

Devex 864: more test cleanup

ericsorenson committed:
Merge branch 'staging' into batch-pretty-momma

codyseibert committed:
8650 - removing the extra space which was added between words that was causing issues with word generated pdfs

kkoskelin committed:
8650: move non-text-search clauses to a filter block

more performant, can be cached.

kkoskelin committed:
8650: revert syntax change until tests are updated

codyseibert committed:
Merge pull request #8754 from flexion/8650-removing-an-extra-space

8650 - removing the extra space which was added between words that wa…

codyseibert committed:
8506 - refresh migrate flag after running setup blue green if needed

kkoskelin committed:
8650: performance boost using filters, with updated tests

codyseibert committed:
8506 - refresh destination table since the script can modify that as well

codyseibert committed:
8506 - getting the migrate and deploy flags are ran later

kkoskelin committed:
8650: enable "OR" and "AND" flags

per journey test

codyseibert committed:
8506 - getting the migrate and deploy flags are ran later

kkoskelin committed:
8650: test fix

enabling OR, AND

codyseibert committed:
Merge pull request #8757 from flexion/8506-refresh-migrate-flag

8506 - refresh migrate flag after running setup blue green if needed

kkoskelin committed:
Merge pull request #8758 from flexion/8650-removing-an-extra-space

8650: performance boost using filters, with updated tests

codyseibert committed:
8684 - fixing unit tests for when flag is true

codyseibert committed:
Merge pull request #8761 from flexion/8684-fixing-tests

8684 - fixing unit tests for when flag is true

ericsorenson committed:
Merge remote-tracking branch 'upstream/staging' into from-staging

ericsorenson committed:
Merge pull request #8765 from flexion/from-staging

From ustaxcourt/staging

ericsorenson committed:
Merge branch 'staging' into batch-pretty-momma

adunkman commented on shared/jest.config.js:
```diff
@@ -16,6 +16,7 @@ module.exports = {
     '!src/**/*_.js',
     '!src/persistence/sqs/deleteMessage.js',
     '!src/persistence/sqs/getMessages.js',
+    '!src/persistence/dynamo/**/*.js',
```
Could you say more about why this should be excluded from coverage? I see a number of files in this folder which are already covered by tests, which would indicate to me that test coverage is desirable here.

ericsorenson replied:
Reverted, that devex item hasn't been entirely completed yet!

adunkman commented on shared/src/business/useCases/updatePetitionerInformationInteractor.getIsUserAuthorized.test.js:
```diff
@@ -0,0 +1,114 @@
+      expect(isUserAuthorized).toBeFalsy();
+    });
+
+    it('should return false when the user is a petitioner attempting to modify another petitioner', async () => {
```
```suggestion
    it('should return false when the user is a petitioner attempting to modify another petitioner', () => {
```

adunkman commented on shared/src/business/useCases/updatePetitionerInformationInteractor.getIsUserAuthorized.test.js:
```diff
@@ -0,0 +1,114 @@
+      expect(isUserAuthorized).toBeFalsy();
+    });
+
+    it('should return true when the user is a petitioner its own contact information', async () => {
```
```suggestion
    it('should return true when the user is a petitioner its own contact information', () => {
```

adunkman commented on shared/src/business/useCases/updatePetitionerInformationInteractor.getIsUserAuthorized.test.js:
```diff
@@ -0,0 +1,114 @@
+      expect(isUserAuthorized).toBeTruthy();
+    });
+
+    it('should return true when the user is representingCounsel', async () => {
```
```suggestion
    it('should return true when the user is representingCounsel', () => {
```

adunkman commented on web-api/migration-terraform/main/lambdas/migration-segments.js:
```diff
@@ -165,6 +153,7 @@ exports.handler = async event => {
     ...(await hasMigrationRan('bug-0036-public-served-parties-code.js')),
     ...(await hasMigrationRan('0036-phone-number-format.js')),
```
Hmm, I see that the code to actually run the migrations was removed above — should these still be here?

ericsorenson replied:
Those check that the migrations have been run in the past, I don't think there's any harm in keeping them there.

adunkman replied:
If we started up a new environment today, it wouldn’t have these migrations ran — and since the migration code has no capability to run these migrations, I don’t believe the code would be deployable. 

Am I missing something? I might be. 🤔

ericsorenson replied:
Updated!

adunkman replied:
Just circling back here to summarize for the future — we discussed this after standup this morning with @mmarcotte and @ericsorenson. 

Meta-level decisions:

- If migrations are removed, they should be deleted from the codebase with its tests, as well as these lines which call `hasMigrationRan`. Removing the `hasMigrationRan` calls at the same time as the migrations keeps the application deployable.
- Since the Court’s environments receive migrations after Flexion’s, @mmarcotte is responsible for removing outdated migrations going forward, and he doesn’t mind being nudged to do so if they’re getting in the way. 
- Removing migration code reduces clutter in this folder and keeps tests running fast. 

For this pull request:

- This PR should be updated to revert removing the other migrations, leaving them unchanged in the PR. 
- The PR should continue to include the introduction of the new migration needed.

adunkman commented on web-api/src/v2/v2ApiWrapper.test.js:
```diff
@@ -20,32 +18,31 @@ describe('v2ApiWrapper', () => {
+  );
+
+  [405, 429, 503].forEach(statusCode =>
+    it(`errors with a status code ${statusCode} not indicated by the v1 spec are 500s`, () =>
```
```suggestion
    it(`errors with a status code ${statusCode} not indicated by the v2 spec are 500s`, () =>
```

adunkman reviewed (commented):
Looks like there are a few introduced linter errors in this pull request — I submitted suggested changes to fix them below, along with two questions.

------

In addition to the stories mentioned in the description, this PR contains: 

- A followup to https://github.com/ustaxcourt/ef-cms/pull/1552 in commit 44b27b4a2a883d7e514d826b856ea5246f7fd64d which removes the initial configuration of `DESTINATION_TABLE` and `MIGRATE_FLAG`.
- https://github.com/flexion/ef-cms/pull/8737
- [Devex 864: Maximum file size for unit tests](https://trello.com/c/TG3luUmS/864-maximum-file-size-for-unit-tests)
   - https://github.com/flexion/ef-cms/pull/8671
   - https://github.com/flexion/ef-cms/pull/8675
   - https://github.com/flexion/ef-cms/pull/8686
   - https://github.com/flexion/ef-cms/pull/8729
   - https://github.com/flexion/ef-cms/pull/8735
   - https://github.com/flexion/ef-cms/pull/8738
   - https://github.com/flexion/ef-cms/pull/8739
   - https://github.com/flexion/ef-cms/pull/8750
   - https://github.com/flexion/ef-cms/pull/8752
- [Devex 971: Boost test coverage to 99% on shared](https://trello.com/c/lYoJDA3y/971-boost-test-coverage-to-99-on-shared)
   - https://github.com/flexion/ef-cms/pull/8677
   - https://github.com/flexion/ef-cms/pull/8742
- [Devex 973: Boost test coverage to 99% on web-api](https://trello.com/c/5DkJY41g/973-boost-test-coverage-to-99-on-web-api)
   - https://github.com/flexion/ef-cms/pull/8709
- [Devex 335: Change camelCased folders to be consistent on web-client](https://trello.com/c/0RcWpGyM/335-change-camelcased-folders-to-be-consistent-on-web-client)
   - https://github.com/flexion/ef-cms/pull/8740

⚠️ Merging this pull request will require running the manual environment-specific terraform step. 

After this pull request is merged, we can update CircleCI configuration through the UI to remove the following variables: 
  - `SONAR_ORG`
  - `SHARED_SONAR_KEY`
  - `SHARED_SONAR_TOKEN`
  - `API_SONAR_KEY`
  - `API_SONAR_TOKEN`
  - `UI_SONAR_KEY`
  - `UI_SONAR_TOKEN`

kkoskelin committed:
Update web-api/src/v2/v2ApiWrapper.test.js

Co-authored-by: Andrew Dunkman (he/him) <adunkman@gmail.com>

kkoskelin committed:
Update shared/src/business/useCases/updatePetitionerInformationInteractor.getIsUserAuthorized.test.js

These linting warnings (and others related) have been addressed in some devex work which will make its way to the court soon.  But we can accept these commit suggestions here also.

Co-authored-by: Andrew Dunkman (he/him) <adunkman@gmail.com>

kkoskelin committed:
Update shared/src/business/useCases/updatePetitionerInformationInteractor.getIsUserAuthorized.test.js

These linting warnings (and others related) have been addressed in some devex work which will make its way to the court soon.  But we can accept these commit suggestions here also.

Co-authored-by: Andrew Dunkman (he/him) <adunkman@gmail.com>

kkoskelin committed:
Update shared/src/business/useCases/updatePetitionerInformationInteractor.getIsUserAuthorized.test.js

These linting warnings (and others related) have been addressed in some devex work which will make its way to the court soon.  But we can accept these commit suggestions here also.

Co-authored-by: Andrew Dunkman (he/him) <adunkman@gmail.com>

ericsorenson committed:
Update jest.config.js

ericsorenson committed:
revert removing old migrations

adunkman commented on web-api/migration-terraform/main/lambdas/migration-segments.js:
```diff
@@ -41,8 +44,8 @@ const migrateRecords = async ({
 }) => {
-  if (!ranMigrations['bug-0035-private-practitioner-representing.js']) {
-    applicationContext.logger.info('about to run bug migration 0035');
+  if (!ranMigrations['bugMigration0035.js']) {
```
I believe this should be testing for `bug-0035-private-practitioner-representing.js` — that’s what this line was before this PR, and I believe your intention was to revert the changes. 

Could you say more about why this should be `bugMigration0035.js` instead?

adunkman commented on web-api/migration-terraform/main/lambdas/migration-segments.js:
```diff
@@ -51,16 +54,21 @@ const migrateRecords = async ({
   }
 
-  if (!ranMigrations['0036-phone-number-format.js']) {
+  if (!ranMigrations['migration0036.js']) {
```
Same as above — I was expecting this to be reverted to `0036-phone-number-format.js`.

adunkman commented on web-api/migration-terraform/main/lambdas/migration-segments.js:
```diff
@@ -51,16 +54,21 @@ const migrateRecords = async ({
+  if (!ranMigrations['migration0036.js']) {
     applicationContext.logger.debug('about to run migration 0036');
-    items = await migration0036(items);
+    items = await migration0036(items, documentClient);
```
Looks like this is also an unintentional change — I don’t believe this migration takes a second argument.

adunkman commented on web-api/migration-terraform/main/lambdas/migrations/0035-in-care-of-to-additional-name.js:
```diff
@@ -0,0 +1,41 @@
+const createApplicationContext = require('../../../../src/applicationContext');
```
This looks like a new migration introduced in this PR — and it doesn’t appear to be called from `migration-segments`.

adunkman commented on web-api/migration-terraform/main/lambdas/migrations/0035-in-care-of-to-additional-name.test.js:
```diff
@@ -0,0 +1,123 @@
+const {
```
Same for this file — I wasn’t expecting this migration (or it’s test) to be introduced here.

adunkman reviewed (changes requested):
I believe there are a few bugs introduced in the revert commit, https://github.com/ustaxcourt/ef-cms/pull/1547/commits/0f9f8316826a5e6ed9531c9d501f7edbdba7cfda — let me know if it’s my misunderstanding!

ericsorenson committed:
address feedback in PR

ericsorenson committed:
Merge remote-tracking branch 'upstream/staging' into batch-pretty-momma

adunkman reviewed (approved):
Looks great — thanks for taking a second look!
